src/interaction/navigationhandler.cpp

```diff
@@ -124,15 +124,25 @@ void NavigationHandler::updateCamera(double deltaTime) {
     if (_cameraUpdatedFromScript) {
         _cameraUpdatedFromScript = false;
     }
-    else if ( ! _playbackModeEnabled ) {
-        if (_camera) {
-            if (_useKeyFrameInteraction) {
-                _keyframeNavigator->updateCamera(*_camera, _playbackModeEnabled);
-            }
-            else {
-                _orbitalNavigator->updateStatesFromInput(*_inputState, deltaTime);
-                _orbitalNavigator->updateCameraStateFromStates(deltaTime);
+    else {
```

Comment:
From discussion yesterday: let's see if we can simplify this logic by calling 
```
_orbitalNavigator->resetVelocities();
_orbitalNavigator->resetNodeMovements();
```
directly when the playback is stopped.